components/src/source.js

```diff
@@ -66,8 +66,8 @@ export const source = {
   'a11y/semantic-enrich': `${src}/a11y/semantic-enrich/semantic-enrich.js`,
   'a11y/complexity': `${src}/a11y/complexity/complexity.js`,
   'a11y/explorer': `${src}/a11y/explorer/explorer.js`,
-  '[sre]': (typeof window === 'undefined' ? `${src}/../../js/a11y/sre-node.js` :
-            `${src}/../../node_modules/speech-rule-engine/lib/sre_browser.js`),
+  'a11y/sre': `${src}/a11y/sre/sre.js`,
+  '[sre]': `${src}/../../es5/sre`,
```

Comment:
This is actually the original location without `source.js`, so there is no reason to set it here.  This line can be removed.